release/__init__.py

```diff
@@ -738,6 +738,15 @@ def create(self, repository_path, channel, tag):
         assert 'complete_dict' in metadata
         assert 'commit' in metadata
 
+        # Assert that each variant's bootstrap's active packages are included in its complete package list.
+        # TODO(branden): Make the complete package list available in the installer (for
+        # dcos_installer.backend.do_aws_cf_configure()) and move this assertion to make_bootstrap_artifacts().
+        for info in metadata['all_completes'].values():
+            bootstrap_active_packages = set(
+                pkgpanda.util.load_json('packages/cache/bootstrap/{}.active.json'.format(info['bootstrap']))
```

Comment:
nit: Will be nice when we can just programatically get back this from `pkgpanda.build_tree` but that's definitely future / out of scope work.